tests/resources/test_design_workflows.py

```diff
@@ -42,24 +48,9 @@ def collection(branch_data, collection_without_branch) -> DesignWorkflowCollecti
 
 
 @pytest.fixture
-def workflow(collection, branch_data, design_workflow_dict) -> DesignWorkflow:
-    design_workflow_dict["branch_root_id"] = branch_data["metadata"]["root_id"]
-    design_workflow_dict["branch_version"] = branch_data["metadata"]["version"]
-
-    collection.session.set_response(branch_data)
-    workflow = collection.build(design_workflow_dict)
-    collection.session.calls.clear()
-
-    workflow.uid = uuid.uuid4()
-    return workflow
-
-
-@pytest.fixture
-def workflow_minimal(collection, workflow) -> DesignWorkflow:
-    workflow.predictor_id = None
-    workflow.predictor_version = None
-    workflow.design_space_id = None
-    return workflow
```

Comment:
This became obsolete because it was only used twice, and so was better served by a Factory with arguments.